README.md

```diff
@@ -1,19 +1,13 @@
+# Upup
+
 ![upup_logo_dark](https://github.com/DevinoSolutions/upup/assets/43147238/b5477db9-cb23-43c7-8c12-518beb31af53)
 
-<br/>
+An open-source, free-to-use NPM component that easily handles your file upload needs with seamless DigitalOcean Spaces, Amazon S3, Google Drive and OneDrive integration.
```

Comment:
Suggestion:

Given that the package relies on JSX and React hooks, it would be more precise to describe it as a "React component library" rather than a generic "NPM component," ensuring the description aligns with its core technology stack.

Reply:
Thanks @BSalaeddin , done